tools/trips_data.json

```diff
@@ -0,0 +1,702 @@
+[
+    {
+        "country": "Austria",
+        "duration": "bardzo kr\u00f3tki",
```

Comment:
why are polish characters like this in .json file? will it look ok in UI?

Reply:
Yes, it should be totally okay. I tested the data with a simple reader script in python and console output had correct characters

Reply:
`ensure_ascii=False` in `json.dump(trips, f, indent=4, ensure_ascii=False)` did the trick